### PR TITLE
Resolved Bug 655 : Billing Address Label issue resolved

### LINKED ISSUE #655: Storybook >Component > Billing Address >Default > Placeholder issue.
**Describe the bug**
After submitting field is not clear.

**Steps to reproduce bug**
1. Go to 'https://stage-react.raaghu.ai/'
2. Navigate to 'Storybook >Component > Billing Address >Default'
3. Fill the data and click on save.

**Actual result-**
After submitting field is not clear.

**Screenshots**
image

**Expected result-**
After submitting all field should be clear.

**Desktop (please complete the following information):**
-OS: [Windows 10 Enterprise]
-Browser [ edge]
-Version [ 125.0.2535.92]

### PR DESCRIPTION
**Pull request by suchithravipin1991**
## Description

1. Address is visible instead of Enter Address

## Screenshot

addess

### CODE REVIEW

File: raaghu-components/src/rds-comp-billing-address/rds-comp-billing-address.tsx [hunk 1]
```diff
@@ -209,7 +209,7 @@ function emitSaveData(event: any) {
                         <div className="col-md-12 mb-3">
                             <RdsInput
                                 label="Address"
-                                placeholder="Address"
+                                placeholder=" Enter Address"
```
Comment (hemalatagaikwad) [inline raaghu-components/src/rds-comp-billing-address/rds-comp-billing-address.tsx]: remove blank space

**Comment by hemalatagaikwad** [comment]
@suchithravipin1991  please add checklist in conversation

**Review by hemalatagaikwad — CHANGES_REQUESTED** [review]

**Review by hemalatagaikwad — APPROVED** [review]
ok

**Review by omkarchoudhari — APPROVED** [review]

**Review by hemalatagaikwad — APPROVED** [review]
ok

**Review by hemalatagaikwad — APPROVED** [review]
ok

**Comment by hemalatagaikwad** [comment]
close this pr as new pr created against this task https://github.com/Wai-Technologies/raaghu-react/pull/799

**Pull request closed without merging**